### DISCUSSION TITLE
gql-tag-operations-preset + VSCode's autocompletions

### DISCUSSION THREAD

**Discussion by kwassholm-work** [dotansimha/graphql-code-generator#6669] [Q&A]
Heya,

I've been trying out the gql-tag-operations-preset as mentioned in https://the-guild.dev/blog/whats-new-in-graphql-codegen-v2. Seems very nice but for some reason autocompletion inside the GraphQL queries does not work anymore in VSCode. I'm using the "graphql.vscode-graphql" plugin.

Any ideas or pointers to get it working? Autocompletion works ok in .graphql files.

**Comment by fivethreeo** [comment]
Got the same problem + https://github.com/dotansimha/graphql-code-generator/issues/6674

**Comment by fivethreeo** [comment]
Make sure you use https://marketplace.visualstudio.com/items?itemName=GraphQL.vscode-graphql

Create graphql.config.json

```json

{
    "schema": ["./src/**/*.{graphql}"],
    "documents": ["./src/**/*.{js,ts,jsx,tsx}"]
  }
```
